scripts/preprocess-docs.js

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env node
+
+/**
+ * Pre-processes Mocha's documentation
+ * 1. Update TOC
+ * 2. Dump `--help` output into "Command-Line Usage" section
+ */
+
+/**
+ * The CLI included with markdown-toc doesn't support `bullets`
+ * and `maxdepth` options, so that's why this exists.
+ */
+
+'use strict';
+
+const toc = require('markdown-toc');
+const miss = require('mississippi');
```

Comment:
Have plans for the other `mississippi` dependencies? Know we use `through2` for browser bundle. Used `pump` and `concat-stream` below. Other 7 unused dependencies worth it?

Reply:
we use `through` like you said, then `concat`, and `pipe` here, and `pipe` uses `finish`.

it's a dev dep.  guess I don't care too much about adding new dev deps.